includes/class-boldgrid-backup.php

```diff
@@ -556,6 +564,9 @@ private function define_admin_hooks() {
 
 		$plugin_notices = new Boldgrid_Backup_Admin_Plugin_Notices();
 		$this->loader->add_filter( 'Boldgrid\Library\Plugin\Notices\admin_enqueue_scripts', $plugin_notices, 'filter' );
+
+		// Get Plugin Update Data for all active plugins.
+		$this->loader->add_filter( 'boldgrid_backup_active_plugins', new \Boldgrid\Library\Library\Plugin\Plugins(), 'getAllPlugins' );
```

Comment:
Instead of defining this here, you can use `@hook something` in the library file (so that other plugins can reuse it). Let me know if you need help doing this.

Reply:
Yeah...I would like to discuss with you how yall have that setup, and whether that would be ideal for this. I created that filter thinking I would use that call more often, but I only end up using that filter once ( admin/class-boldgrid-backup-premium-admin-plugins.phpL#58 ) , so it may be just as effective to not have that filter at all

Reply:
> I would like to discuss with you how yall have that setup
The system for adding hooks via doc blocks in the library has been around since the library was launched. It's heavily integrated, and there's no plans to make any changes right now to it.

Reply:
I got rid of the 'boldgrid_backup_active_plugins' filter since it was not really needed. I created that when i thought I would need to use it in more places. Since I do not, I got rid of it.